Update OSD throttle options

MrD-RC opened the pull request:
Fixes #8801 

Pre-INAV 3.0 there were two working throttle display elements. A PR in 2.7 (#6510) made a change that meant both OSD throttle elements showed the same data. This is pretty pointless. The previous method allowed the display of the throttle in a non-scaled view, which made selecting navigational throttles easier. This PR allows the pilot to see scaled throttle, unscaled, or both. Both throttles work for manual and auto throttle.

Configurator https://github.com/iNavFlight/inav-configurator/pull/1725

MrD-RC committed:
Update OSD throttle options

Pre-INAV 3.0 there were two working throttle display elements. A PR in 2.7 made a change that meant both OSD throttle elements showed the same data. This is pretty pointless. The previous method allowed the display of the throttle in a non-scaled view, which made choosing navigational throttles easier. This PR allows the pilot to see scaled throttle, unscaled, or both. Both throttles work for manual and auto throttle.

breadoven commented on src/main/flight/mixer.c:
```diff
@@ -609,9 +609,15 @@ void FAST_CODE mixTable()
+    int16_t thr = constrain(rcCommand[THROTTLE], PWM_RANGE_MIN, PWM_RANGE_MAX); 
+    
+    if (useScaled) {
+        thr = (thr - getThrottleIdleValue()) * 100 / (motorConfig()->maxthrottle - getThrottleIdleValue());
```
`getThrottleIdleValue `should be assigned to a const

MrD-RC replied:
True. No point in multiple calls.

breadoven commented on src/main/flight/mixer.c:
```diff
@@ -609,9 +609,15 @@ void FAST_CODE mixTable()
+    if (useScaled) {
+        thr = (thr - getThrottleIdleValue()) * 100 / (motorConfig()->maxthrottle - getThrottleIdleValue());
+    } else {
+        thr = (thr - 100) / 10;
```
1000 surely ?

tiriad replied:
thr = (thr - PWM_RANGE_MIN) * 100 / (PWM_RANGE_MAX - PWM_RANGE_MIN);

MrD-RC replied:
@breadoven Yes, should be -1000, typo.

@tiriad Actually, this would be better, as it is based on PWM_RANGE_MIN and PWM_RANGE_MAX.

tiriad replied:
Yes, I only suggested it to follow the style of other variables and the code prior to the modification assuming it was correct. https://github.com/iNavFlight/inav/blob/5510259ec3fb255dbf1a14b6660adb2aaf50679e/src/main/io/osd.c#L879

MrD-RC committed:
Fix typo and use const for idle throttle

MrD-RC committed:
Update non-scaled to use PWM MIN/MAX rather than values

MrD-RC committed:
Make telemetry mirror OSD

MrD-RC committed:
Updates after testing

MrD-RC committed:
Update throttle symbols

Rather than having a `*` prefix to indicate the PWM throttle. I'm now using the scale symbol (also used in the map) to prefix the scaled throttle. This makes a lot more sense.
